src/ROUTES.ts

```diff
@@ -37,16 +36,9 @@ const ROUTES = {
     ALL_SETTINGS: 'all-settings',
 
     SEARCH_CENTRAL_PANE: {
-        route: '/search/:query',
-        getRoute: (searchQuery: SearchQuery, queryParams?: AuthScreensParamList['Search_Central_Pane']) => {
-            const {sortBy, sortOrder} = queryParams ?? {};
-
-            if (!sortBy && !sortOrder) {
-                return `search/${searchQuery}` as const;
-            }
-
-            return `search/${searchQuery}?sortBy=${sortBy}&sortOrder=${sortOrder}` as const;
-        },
+        route: 'search',
+        getRoute: ({query, isCustomQuery = false, policyIDs}: {query: SearchQueryString; isCustomQuery?: boolean; policyIDs?: string}) =>
```

Comment:
Adam is back on Tuesday next week, and I simply don't know why this is needed, so we need to wait for him to add the comment.
Perhaps we could merge without it, dunno yet I will ask @luacmartins

Reply:
This is needed for filters since selecting any filter will navigate the user to the Search Results page that's being implemented [here](https://github.com/Expensify/App/pull/45409). I know it's a bit out of place now, but we should just leave this in place.